src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,69 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io. FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFileName(toFileName, optimizationTextCsv(getTextFromFile(fromFileName)));
+    }
+
+    public String getTextFromFile(String fromFileName) {
+        StringBuilder readTextBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String readText = bufferedReader.readLine();
+            while (readText != null) {
+                String newReadText = readText.replace(",", " ");
+                readTextBuilder.append(newReadText).append(System.lineSeparator());
+                readText = bufferedReader.readLine();
+            }
+
+        } catch (IOException e) {
+            throw new RuntimeException("not is good read file", e);
+        }
+        return String.valueOf(readTextBuilder);
+    }
+
+    public String optimizationTextCsv(String getTextFromFile) {
+        StringBuilder readTextBuilder = new StringBuilder(getTextFromFile);
+        long supply = 0;
+        long buy = 0;
+        long result = 0;
 
+        String[] optimizationText = String.valueOf(readTextBuilder).split(System.lineSeparator());
+        for (String cloud : optimizationText) {
+            final int CutIndexText = 0;
+            final int CutIndexNumber = 1;
```

Comment:
make those variable constants, make them global, 
private static final 
and choose appropriate names OPERATION_INDEX, AMOUNT_INDEX

Reply:
fixed